Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "euclid"
-version = "0.17.0"
+version = "0.18.0"
```

Comment:
This isn't a breaking change, so i'd prefer to make it `0.17.1` instead of `0.18.0`.